utils/uniset/src/builder.rs

```diff
@@ -310,7 +310,7 @@ impl UnicodeSetBuilder {
     #[allow(unused_assignments)]
     pub fn retain_set(&mut self, set: &UnicodeSet) {
         let mut prev = 0;
-        for pair in set.as_inversion_list().as_slice().chunks(2) {
+        for pair in set.as_inversion_list().as_ule_slice().chunks(2) {
```

Comment:
Note: cases where we use `.chunks()` or `.windows()` need an actual slice.  So this is a reason to keep `as_ule_slice()` public.